supplementary/alica_ros1/alica_tests/libalica-tests/CMakeLists.txt

```diff
@@ -0,0 +1,43 @@
+cmake_minimum_required(VERSION 3.5.1)
+project(libalica-tests)
+
+include(${CMAKE_CURRENT_SOURCE_DIR}/../../../../cmake_flags/cflags.cmake)
+alica_compile_flags()
+
+add_library(${PROJECT_NAME} SHARED
```

Comment:
If we just statically link all of these rather than making a new library will it work?

Reply:
No that doesnt work with the current implementation. If we want to use a mixture of dynamically linked objects from the alica_standard_lib and of project dependent elements (alica_tests) we have to update the dynamic creators, because they will look for a dynamically linked lib with the given lib name and throw an error if they cant find one. I tried it with supplementary_tests:

```
[ERROR][TEST][DynamicLoading]: Error:Lib not exists in LD_CONFIG_PATH library name:libalica-supplementary-tests

[ERROR][TEST][DynamicLoading]: Could not compute the complete library name for creating the behaviour: AcmeBeh
```